content/docs/2.5/scalers/elasticsearch.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Elasticsearch"
+availability = "v2.5+"
+maintainer = "Community"
+description = "Scale applications based on elasticsearch search template query result."
+layout = "scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `elasticsearch` trigger that scales based on result of an [elasticsearch search template](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-template.html) query. This lets you run searches without exposing Elasticsearch’s query syntax to the keda scaler.
```

Comment:
```suggestion
This specification describes the `elasticsearch` trigger that scales based on the result of an [elasticsearch search template](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-template.html) query.

This lets you run searches without exposing Elasticsearch’s query syntax to the KEDA scaler.
```

Reply:
Does the last line actually matter to end-users?

Reply:
No. This can be removed.